integration-tests/actions/ocr_helpers.go

```diff
@@ -228,16 +228,13 @@ func CreateOCRJobs(
 				Name: nodeContractPairID,
 				URL:  fmt.Sprintf("%s/%s", mockserver.Config.ClusterURL, strings.TrimPrefix(nodeContractPairID, "/")),
 			}
-			// only create the bridge on the first loop of the nodes
-			if i < 1 {
-				err = SetAdapterResponse(mockValue, ocrInstance, node, mockserver)
-				if err != nil {
-					return fmt.Errorf("setting adapter response for OCR node failed: %w", err)
-				}
-				err = node.MustCreateBridge(bta)
-				if err != nil {
-					return fmt.Errorf("creating bridge job have failed: %w", err)
-				}
+			err = SetAdapterResponse(mockValue, ocrInstance, node, mockserver)
```

Comment:
Guessing this does not apply anymore right?-

> // only create the bridge on the first loop of the nodes

Reply:
Ya, removing this on next PR, missed here